src/frauddetectionservice/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:18-slim AS builder
```

Comment:
openjdk is deprecated: https://hub.docker.com/_/openjdk

Please replace it with https://hub.docker.com/_/eclipse-temurin
Also, Java 18 is not LTS, please use Java 17.